spec/01-abstract.md

```diff
@@ -1,19 +1,39 @@
-## Problem Statement
+# Problem Statement
 
-Distributed tracing is a methodology implemented by tracing tools enabling to follow, analyze and debug a transaction across multiple software components. Typically, a transaction passes more than one component which requires a transaction to be uniquely identified across all software components. Passing along this unique identification is referred to as context propagation.
+Distributed tracing is a methodology implemented by tracing tools to follow, analyze
+and debug a transaction across multiple software components. Typically, a trace
+traverses more than one component which requires it to be uniquely identifiable
+across all participating systems. Passing along this unique identification is
+referred to as *context propagation*.
 
-Today context propagation is implemented in proprietary formats individually by each tracing tool provider. This results in siginficant interoperability problems between tracing tools than manifest in two major ways:
+Today, context propagation is implemented individually by each vendor.
+In multi-vendor environments, this causes interoperability problems, like:
 
-- Traces that are collected by two tracing providers cannot be linked together as there is no unique identifier linking the individual pieces together. 
-- Traces are broken each time they are passed between components monitored by different tracing as there is no uniformly agree set of identifcation taht is forwarded.
+- Traces that are collected by different tracing vendors cannot be linked together
+- as there is no shared unique identifier.

```

Comment:
this is not supposed to be a separate bullet, is it?

Reply:
Fixed in https://github.com/w3c/trace-context/pull/224/commits/951a7c920882acefb88bd0286d5549df9c3382c6